iroha_p2p/src/peer.rs

```diff
@@ -109,27 +140,21 @@ where
     }
 
     async fn handshake(&mut self) -> Result<(), Error> {
+        debug!("Attempting handshake in state {:?}...", &self.state);
```

Comment:
Better to have logs in some manner like this:
```rust
let state = self.state;
debug!(state, "Attempting handshake");
```

You can look at tracing library

Reply:
You can also use spans

Reply:
Why is is better? In my manner it is more readable, I think.

Reply:
It is about log sorting in future. It can be used for structured grepping